libr/core/cmd_zign.c

```diff
@@ -292,112 +289,39 @@ static bool addBytesZign(RCore *core, const char *name, int type, const char *ar
 	return retval;
 }
 
-static bool addOffsetZign(RCore *core, const char *name, const char *args0, int nargs) {
-	const char *offstr = NULL;
-	ut64 offset = UT64_MAX;
-
-	if (nargs != 1) {
+static bool addOffsetZign(RCore *core, const char *name, RList *args) {
+	if (r_list_length (args) != 1) {
 		eprintf ("error: invalid syntax\n");
 		return false;
 	}
-
-	offstr = r_str_word_get0 (args0, 0);
-	offset = r_num_get (core->num, offstr);
-
+	const char *offstr = (const char *)r_list_get_top (args);
+	ut64 offset = r_num_get (core->num, offstr);
```

Comment:
check for null... you are checking for list length first.. but i think lists can be filled with nulls.